src/gmp/models/event.js

```diff
@@ -380,8 +380,9 @@ class Event {
       // the event is not recurring
       // it should only occur once on its start date
       const now = date();
+      const start = convertIcalDate(this.event.startDate, this.timezone);
 
-      if (this.event.startDate.toUnixTime() >= now.unix()) {
+      if (start.unix() >= now.unix()) {
         return convertIcalDate(this.event.startDate, this.timezone);
```

Comment:
this is the same as start right?
```suggestion
        return start;
```